QMLComponents/variableinfo.cpp

```diff
@@ -68,5 +70,12 @@ bool VariableInfo::dataAvailable()
 
 DataSet *VariableInfo::dataSet()
 {
-	return _provider ? reinterpret_cast<DataSet*>(_provider->provideInfo(VariableInfo::DataSetPointer).value<void*>()) : nullptr;
+	static DataSet* singleDataSet = nullptr;
```

Comment:
This is not going to work is it?
How do you reflect the dataset getting destroyed?

Reply:
Yeah, so as I suspected:
- open dataset, use a sampling audit analysis, it works
- close dataset, dont close jasp
- open other dataset, use a sampling audit analysis, its broken...

Ill just make sure the datasetpointer works, or Ill fix it some other way

Reply:
Hmm I forgot this case.